resources/assets/sass/components/_tags.scss

```diff
@@ -0,0 +1,46 @@
+// Tags
+//
+// Tags component
+//
+// Markup:
+//<ul class="c-tags">
+//<li class="c-tags__item {$modifiers}"><a href="#" class="c-tags__item-link">Tag name</a></li>
+//<li class="c-tags__item {$modifiers}"><a href="#" class="c-tags__item-link">Tag name</a></li>
+//</ul>
+//
+// .m-yellow - Yellow
+// .m-green - Green
+// .m-red - Red
+// .m-blue - Blue
+// .m-purple - Purple
+// .m-orange - Orange
+//
+// Styleguide 10
+
+.c-tags {
+
+	@include clearfix;
+	@include clear-list;
+	@include font-smoothing;
+}
+
+	.c-tags__item {
+
+		@include background-color-modifiers;
+
+		display: inline-block;
+		border-radius: 12px;
+	}
+
+		.c-tags__item-link {
+
+			color: $color-white;
+			font-family: $font-primary;
```

Comment:
Properties should be ordered color, display, font-family, font-size, font-weight, height, line-height, padding, text-decoration